test/common/json/json_loader_test.cc

```diff
@@ -445,6 +445,13 @@ TEST(JsonLoaderTest, YamlObject) {
   }
 }
 
+TEST(JsonLoaderTest, YamlAsJsonString) {
+  {
```

Comment:
nit if you feel like it: I would remove brace at 449/452. They are not needed as this is the only thing that happens in this test.

Reply:
done!